src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -350,15 +369,15 @@ private void SendNewReleaseRoleEmail(Guid userId, IdTitlePair publication, IdTit
 
             if (existing != null)
             {
-                return ValidationActionResult(UserAlreadyHasReleaseRole);
+                return ValidationActionResult(ValidationErrorMessages.UserAlreadyHasReleaseRole);
```

Comment:
I tend to prefer the static import way of supplying the error messages - less verbose and I'd say it's obvious what's happening.  I guess it's a question of taste - we should probably have a developer moot some time and pick a strategy!